c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +341,18 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
 
 
 /**
+ * @param {appx.Document} data Document attributes.
+ * @return {appx.Document}
+ * @export
```

Comment:
Indeed, it's only that the method can be overridden in the child controllers.
Same fix: removing the ``@export`` tag?